server/spec/mutations/stacks/create_spec.rb

```diff
@@ -23,6 +23,31 @@
 
     it 'creates a new grid stack with parent' do
       grid
+      grid.stacks.create!(name: 'stack-parent')
+      expect {
+        outcome = described_class.new(
+          grid: grid,
+          name: 'stack',
+          stack: 'foo/bar',
+          version: '0.1.0',
+          registry: 'file://',
+          source: '...',
+          variables: {foo: 'bar'},
+          services: [{name: 'redis', image: 'redis:2.8', stateful: true }],
+          parent: {
+            id: "#{grid.name}/stack-parent"
+          }
+        ).run
+
+        expect(outcome).to be_success
+        expect(outcome.result.parent_name).to eq 'stack-parent'
+        expect(outcome.result.has_parent?).to be_truthy
+      }.to change{ Stack.count }.by(1)
+    end
+
+    it 'creates a new grid stack with deprecated parent_name' do
+      grid
+      grid.stacks.create!(name: 'stack-parent')
```

Comment:
Creating the parent stack as part of the spec breaks the specs (false-positive).